Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/05-Network-LZ/lz-networking.tf

```diff
@@ -5,33 +5,25 @@ resource "azurerm_resource_group" "spoke-rg" {
   location = data.terraform_remote_state.existing-hub.outputs.hub_rg_location
 }
 
-output "lz_rg_location" {
-  value = azurerm_resource_group.spoke-rg.location
-}
-
-output "lz_rg_name" {
-  value = azurerm_resource_group.spoke-rg.name
-}
-
-
 # Virtual Network
 
 resource "azurerm_virtual_network" "vnet" {
   name                = "${var.lz_prefix}-vnet"
   resource_group_name = azurerm_resource_group.spoke-rg.name
   location            = azurerm_resource_group.spoke-rg.location
-  address_space       = ["10.1.0.0/16"]
-  dns_servers         = null
+  address_space       = ["10.240.0.0/16"]
+  dns_servers         = ["10.200.0.100"]
   tags                = var.tags
 
 }
 
-output "lz_vnet_name" {
-  value = azurerm_virtual_network.vnet.name
-}
+resource "azurerm_subnet" "priv-link" {
+  name                                           = "priv-subnet"
```

Comment:
Naming is still inconsistent.  Even in this vnet, the subnets do not share a consistent naming.

Reply:
Captured in #53